.github/workflows/compilation_on_windows.yml

```diff
@@ -39,6 +39,12 @@ on:
   # allow to be triggered manually
   workflow_dispatch:
 
+env:
+  # For Spec Test
+  DEFAULT_TEST_OPTIONS: "-s spec -b"
```

Comment:
Can we use `-P` option to run the spec test parallelly?

Reply:
There's something in Python script that won't work when enabling parallel execution. I'm yet to investigate it but I thought we can enable those tests in a single-threaded mode.

Reply:
Got it, thanks.